web/terraform/variables.tf

```diff
@@ -58,6 +61,7 @@ variable "configuration" {
       zone_balancing_enabled = false
       worker_count           = 1
       front_door_sku_name    = "Standard_AzureFrontDoor"
+      waf_mode               = "Prevention"
```

Comment:
How confident are we that no legitimate user has an IP that Azure thinks is outside the UK?

Reply:
Discussed offline; marking as resolved.

`block-invalid-request-method` added to ideally block malicious traffic before reaching this rule.